metaflow/plugins/argo/argo_workflows.py

```diff
@@ -143,6 +143,37 @@ def _sanitize(name):
         # allowed by Metaflow - guaranteeing uniqueness.
         return name.replace("_", "-")
 
+    @classmethod
+    def remove_schedule(cls, name):
+        client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
+
+        try:
+            response = client.delete_workflow_schedule(name)
+        except Exception:
+            raise ArgoWorkflowsException(repr(e))
+        if response is None:
+            return (
+                False  # Failing to remove the schedule is not a show stopper for now.
+            )
+
+        return True
+
+    @classmethod
+    def delete(cls, name):
+        client = ArgoClient(namespace=KUBERNETES_NAMESPACE)
+
+        try:
+            response = client.delete_workflow_template(name)
+        except Exception as e:
+            raise ArgoWorkflowsException(repr(e))
+        if response is None:
+            raise ArgoWorkflowsException(
+                "The workflow *%s* doesn't exist "
```

Comment:
This messaging feels a bit weird.  I don't think users want to fix the deletion error by first deploying it :)